release.Rmd

```diff
@@ -26,21 +63,304 @@ To get your package ready to release, follow these steps:
+
+Here we explain ways to do reverse dependency checks at scale, which is the problem we face.
+Some of the packages maintained by our team have thousands of reverse dependencies and even some of the lower-level packages have hundreds.
+We have to approach this in an automated fashion and this section will be most useful to other maintainers in the same boat.
+
+All of our reverse dependency tooling is concentrated in the revdepcheck package (<https://revdepcheck.r-lib.org/>).
+Do this when you're ready to do revdep checks for the first time:
+
+```{r}
+#| eval: false
+usethis::use_revdep()
+```
+
+This does some one-time setup in your package's `.gitignore` and `.Rbuildignore` files.
+You will also see this reminder to actually perform revdep checks like so, as the checklist item suggests:
+
+```{r}
+#| eval: false
+revdepcheck::revdep_check(num_workers = 4)
+```
+
+This runs `⁠R CMD check⁠` on all of your reverse dependencies, with our recommendation to use 4 parallel workers to speed things along.
+The output looks something like this:
+
+    > revdepcheck::revdep_check(num_workers = 4)
+    ── INIT ───────────────────────────────────── Computing revdeps ──
+    ── INSTALL ───────────────────────────────────────── 2 versions ──
+    Installing CRAN version of cellranger
+    also installing the dependencies 'cli', 'glue', 'utf8', 'fansi', 'lifecycle', 'magrittr', 'pillar', 'pkgconfig', 'rlang', 'vctrs', 'rematch', 'tibble'
+
+    Installing DEV version of cellranger
+    Installing 13 packages: rlang, lifecycle, glue, cli, vctrs, utf8, fansi, pkgconfig, pillar, magrittr, tibble, rematch2, rematch
+    ── CHECK ─────────────────────────────────────────── 8 packages ──
+    ✔ AOV1R 0.1.0                     ── E: 0     | W: 0     | N: 0
+    ✔ mschart 0.4.0                   ── E: 0     | W: 0     | N: 0
+    ✔ googlesheets4 1.0.1             ── E: 0     | W: 0     | N: 1
+    ✔ readODS 1.8.0                   ── E: 0     | W: 0     | N: 0
+    ✔ readxl 1.4.2                    ── E: 0     | W: 0     | N: 0
+    ✔ readxlsb 0.1.6                  ── E: 0     | W: 0     | N: 0
+    ✔ unpivotr 0.6.3                  ── E: 0     | W: 0     | N: 0
+    ✔ tidyxl 1.0.8                    ── E: 0     | W: 0     | N: 0                  
+    OK: 8                                                                                 
+    BROKEN: 0
+    Total time: 6 min
+    ── REPORT ────────────────────────────────────────────────────────
+    Writing summary to 'revdep/README.md'
+    Writing problems to 'revdep/problems.md'
+    Writing failures to 'revdep/failures.md'
+    Writing CRAN report to 'revdep/cran.md'
+
+To avoid false positives, `revdep_check()` runs `⁠R CMD check⁠` twice per revdep: once with the released version of your package currently on CRAN and again with the local development version, i.e. with your release candidate.
```

Comment:
```suggestion
To avoid false positives as much as possible, `revdep_check()` runs `⁠R CMD check⁠` twice per revdep: once with the released version of your package currently on CRAN and again with the local development version, i.e. with your release candidate.
```

Reply:
I went with "To minimize false positives".